notebooks/OrionDBExplorer Tutorial.ipynb

```diff
@@ -462,9 +455,9 @@
     }
    ],
    "source": [
-    "from orion import get_available_templates\n",
+    "from orion.analysis import get_available_templates\n",
     "\n",
-    "get_available_templates()"
+    "get_available_templates(None)"
```

Comment:
Why we need to pass None here?

Reply:
That was a test from my end, I removed it